### PR TITLE
docs: add generate-dependabot-glob-action in readme

### PR DESCRIPTION
**Pull request by giacomoaluigi**
- [README](https://github.com/Alfresco/alfresco-build-tools/blob/master/docs/README.md) updated after adding/changing behaviour of an action

### Description
Add generate-dependabot-glob-action documentation to _GitHub Actions provided by community_ section.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -314,6 +315,32 @@ on:
...
+      interval: 'daily'
+  - package-ecosystem: 'docker'
+    directory: '/test/docker/container_2/'
+    schedule:
+      interval: 'daily'
+  - package-ecosystem: 'docker'
+    directory: '/test/docker/weird_dockerfile/'
+    schedule:
+      interval: 'daily'
+```
```
Comment (gionn) [inline docs/README.md:342]:
```suggestion
For example, the following template:

```
  - package-ecosystem: 'docker'
    directory: '/test/docker/*/Dockerfile*'
    schedule:
      interval: 'daily'
```

Will result in:

```
  - package-ecosystem: 'docker'
    directory: '/test/docker/container_1/'
    schedule:
      interval: 'daily'
  - package-ecosystem: 'docker'
    directory: '/test/docker/container_2/'
    schedule:
      interval: 'daily'
  - package-ecosystem: 'docker'
    directory: '/test/docker/weird_dockerfile/'
    schedule:
      interval: 'daily'
```
```

File: docs/README.md [hunk 2]
```diff
@@ -28,6 +28,7 @@ Here follows the list of GitHub Actions topics available in the current document
     - [Retry failing step](#retry-failing-step)
     - [SSH debug](#ssh-debug)
     - [Triggering a workflow in another repository](#triggering-a-workflow-in-another-repository)
+    - [Generate Dependabot Glob Action](#generate-dependabot-glob-action)
```
Comment (atchertchian) [inline docs/README.md:31]: Maybe it's not obvious but there are ordered alphabetically: do you mind moving it please?

File: docs/README.md [hunk 3]
```diff
@@ -314,6 +315,36 @@ on:
   workflow_dispatch:
 ```
 
+### Generate Dependabot Glob Action
+
+[generate-dependabot-glob-action](https://github.com/Makeshift/generate-dependabot-glob-action) creates a dependabot.yml file from a user-provided template by replacing instances of directory globs with an array of objects matching that glob, with all the other keys copied.
+
+For example, the following template:
+
+```
```
Comment (atchertchian) [inline docs/README.md:324]:
```suggestion
```yml
```

(this is making pre-commit fail)

File: docs/README.md [hunk 4]
```diff
@@ -314,6 +315,36 @@ on:
...
+  - package-ecosystem: 'docker'
+    directory: '/test/docker/*/Dockerfile*'
+    schedule:
+      interval: 'daily'
+```
+
+Will result in:
+
+```
```
Comment (atchertchian) [inline docs/README.md:333]:
```suggestion
```yml
```

(this is making pre-commit fail)

**Review by atchertchian — CHANGES_REQUESTED** [review]
See minor comments to address

**Review by atchertchian — APPROVED** [review]
LGTM!

**Pull request merged**